src/unix/process.c

```diff
@@ -1063,9 +1063,14 @@ int uv_process_kill(uv_process_t* process, int signum) {
 
 
 int uv_kill(int pid, int signum) {
-  if (kill(pid, signum))
+  if (kill(pid, signum)) {
+#if defined(__MVS__)
+    /* EPERM is returned if the process is a zombie. */
+    if (errno == EPERM && getpgid(pid) == getpgid(0))
```

Comment:
Would it be more accurate to test exactly if the process is still alive at this point?
```suggestion
    if (errno == EPERM && waitid(P_PID, pid, &infop, WNOHANG | WNOWAIT) == 0)
```

Reply:
Makes sense, I have adjusted the code accordingly.